runbotics/runbotics-orchestrator-ui/src/src-app/components/Tile/ProcessCollectionTile/ProcessCollectionTile.styles.ts

```diff
@@ -10,9 +10,18 @@ export const ProcessCollectionTileWrapper = styled(Paper)(({ theme }) => `
         max-width: 350px;
         padding-block: 2px;
         background-color: ${theme.palette.grey[200]};
+        box-shadow: 0 0.25rem 0.5rem 0 ${theme.palette.grey[400]};
```

Comment:
the whole shadow should be from theme
in theme define e.g.

theme.shadow.card: 0 0.25rem 0.5rem 0 ${theme.palette.grey[400]};

and use it later as just theme.shadow.card in both processTile and processCollectionTile